remake.yml

```diff
@@ -85,6 +79,15 @@ targets:
 
   gfdl_config:
     command: load_config("configs/GFDL_config.yml")
+    
+  ipsl_config:
+    command: load_config("configs/IPSL_config.yml")
```

Comment:
The Notaro configs are all so similar that I would like to eventually just use one config for this, but not a priority now and maybe we'll just keep as-is